src/core/Global.ttslua

```diff
@@ -127,10 +130,25 @@ local overallStats = {
 -- general code
 ---------------------------------------------------------
 
-function onLoad()
+-- saving state of optionPanel to restore later
+function onSave() return JSON.encode(optionPanel) end
+
+function onLoad(savedData)
+  if savedData then
+    optionPanel = JSON.decode(savedData)
+    for id, enabled in pairs(optionPanel) do
+      if enabled then self.UI.setAttribute("toggle" .. id, "isOn", true) end
+    end
+  else
+    for i = 1, 8 do
```

Comment:
Hardcoding like this will mean it has to change every time an option gets added.  It's also going to get messy if/when we get an option that isn't just a boolean.  Better to have the options handle nil values gracefully.